cit-api/pipeline/migrations/0118_auto_20210210_2246.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 2.2.16 on 2021-02-10 22:46
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+def populate_statuses(apps, schema_editor):
+    ApprovalStatus = apps.get_model("pipeline", "ApprovalStatus")
+    statuses = [
+        ["Pending Review", "Opportunity has been submitted and is awaiting approval."],
+        ["Needs to be edited", "Opportunity has been sent back for editing."],
+        ["Not completed", "Opportunity has been saved by submitter in an incomplete state."],
+        ["Published", "Opportunity has been approved and is now publicly viewable."]
+    ]
+    for status in statuses:
+        approval_status = ApprovalStatus()
+        approval_status.status_name = status[0]
+        approval_status.status_description = status[1]
+        approval_status.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pipeline', '0117_auto_20210210_2148'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ApprovalStatus',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('status_name', models.CharField(max_length=255)),
+                ('status_description', models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='opportunity',
+            name='approval_status',
+            field=models.ForeignKey(default=1, on_delete=django.db.models.deletion.CASCADE, to='pipeline.ApprovalStatus'),
```

Comment:
I don't think you want a cascade on this.

Reply:
Good call, I'll fix this